controllers/feeds/getFeedChat.js

```diff
@@ -222,7 +236,12 @@ const getFeedChatService = async (req, res) => {
         actor,
         showToast = false
       } = getDetail(req, b, myAnonymousId.user_id);
+
+      const user = karmaScores.find((user) => user.user_id === actor.id);
+      actor.karmaScores = roundingKarmaScore(user?.karma_score || 0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The karma score for each actor is fetched and rounded off before being assigned to the `actor.karmaScores` property. This ensures that the karma score is always up-to-date and rounded off before being used.
```diff
+ const user = karmaScores.find((user) => user.user_id === actor.id);
+ actor.karmaScores = roundingKarmaScore(user?.karma_score || 0);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->